docs/src/main/sphinx/connector/hive.rst

```diff
@@ -339,9 +339,14 @@ Hive configuration properties
     * - ``hive.max-partitions-per-writers``
       - Maximum number of partitions per writer.
       - 100
+    * - ``hive.max-partitions-for-eager-load``
+      - The maximum number of partitions for a single table scan to haven loaded
+        eagerly on the coordinator. Certain optimizations are not possible
+        without eager loading.
+      - 100,000
     * - ``hive.max-partitions-per-scan``
       - Maximum number of partitions for a single table scan.
-      - 100,000
+      - 1,000,000
```

Comment:
1000000

Reply:
This is consistent with all other properties on this table, and this is more readable considering the number of zeroes.